src/app/workflow/workflow.component.html

```diff
@@ -170,16 +170,25 @@ <h3>
           </mat-tab>
           <div *ngIf="entryType === EntryType.BioWorkflow">
             <mat-tab id="launchTab" label="Launch">
-              <app-launch
-                *ngIf="workflow?.workflowVersions.length > 0; else noVersions"
-                [selectedVersion]="selectedVersion"
-                [basePath]="workflow?.path"
-                [path]="workflow?.full_workflow_path"
-                [canRead]="canRead"
-                [canWrite]="canWrite"
-                [isOwner]="isOwner"
-                [mode]="workflow?.mode"
-              ></app-launch>
+              <div *ngIf="!(launchSupport$ | async); else launchSupported">
+                <div class="p-3">
+                  <mat-card class="alert alert-warning" role="alert">
+                    <mat-icon>warning</mat-icon> Dockstore does not support launching with this language.
```

Comment:
Seems a bit too final. 
Maybe `Dockstore does not yet have customized launch-with instructions for X language.` Ideas welcome

Reply:
"Launch-with support for X language coming soon see [link to documentation?] for more details"

It can lead to the documentation page where we talk about the roadmap or how to manually download and upload to galaxy in the interim?

Reply:
Documentation page for the other languages is not going to work.